docs/preview/features/logging.md

```diff
@@ -61,6 +61,57 @@ app.UseEndpoints(endpoints => endpoints.MapControllers());
 If you have multiple middleware components configured, it is advised to put the `ExceptionHandlingMiddleware` as soon as possible.
 By doing so, unhandled exceptions that might occur in other middleware components will also be logged by the `ExceptionHandlingMiddleware` component.
 
+### Configuration
+
+When custom exception handling is required, you can inherit from the `ExceptionHandlingMiddleware` to create your own middleware component and register it with Arcus' extension.
+
+This example implements the exception handling middleware to influence the log message and returns a custom determination of the HTTP response status code.
+
+```csharp
+using Arcus.WebApi.Logging;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+
+public class MyExceptionHandlingMiddleware : ExceptionHandlingMiddleware
+{
+    public MyExceptionHandlingMiddleware(RequestDelegate next) : base(next)
+    {
+    }
+
+    protected override void LogException(ILoggerFactory loggerFactory, Exception exception)
+    {
+        ILogger logger = loggerFactory.CreateLogger<MyExceptionHandlingMiddleware>();
+        logger.LogCritical(exception, "Custom exception handling!");
+    }
+
+    protected override HttpStatusCode DetermineResponseStatusCode(Exception exception)
```

Comment:
Instead of just allowing to customize the http status code that is returned, I think we should allow end-users (developers) to completely customize the error - response (not only the status code, but also the body)

Reply:
Aha! Yeah, that could be useful. Also having access to the returned headers and all. 👍

Reply:
Done in https://github.com/arcus-azure/arcus.webapi/pull/342/commits/08ede01488cef27e726f8cea3ed343379cce1ed3